package.json

```diff
@@ -1212,7 +1212,7 @@
     "marked": "^0.8.0",
     "moment": "^2.22.1",
     "node-emoji": "^1.8.1",
-    "node-fetch": "^2.1.1",
+    "node-fetch": "v3.0.0-beta.7",
```

Comment:
I had to go for the specifc version since doing ^ caused it to pull `3.0.0-beta.6-exportfix`, which doesn't work for other reasons.